Libraries/WebSocket/RCTWebSocket.xcodeproj/project.pbxproj

```diff
@@ -223,7 +223,7 @@
 				MTL_ENABLE_DEBUG_INFO = YES;
 				ONLY_ACTIVE_ARCH = YES;
 				SDKROOT = iphoneos;
-				SKIP_INSTALL = YES;
+				USER_HEADER_SEARCH_PATHS = "$(SRCROOT)/../Blob";
```

Comment:
Undo. Instead configure the copy files/headers phase correctly, or specify the dependencies correctly in xcode.